packages/preview-middleware-client/src/adp/api-handler.ts

```diff
@@ -86,7 +87,7 @@ export async function request<T>(endpoint: ApiEndpoints, method: RequestMethod,
                 return response.json() as T;
         }
     } catch (e) {
-        throw new Error(e.message);
+        throw new Error((e as Error)?.message);
```

Comment:
Why are we doing this?

Reply:
Fixed in [39505b0](https://github.com/SAP/open-ux-tools/pull/1918/commits/39505b01b7d9cb4ccdc43b1ab21387e06780dc7a)